File: Deliverables/BizBong/app/src/main/java/gamesoftitalia/bizbong/entity/Classifica.java
package gamesoftitalia.bizbong.entity;

import java.util.List;

/**
 * Created by GameSoftItalia on 09/01/2017.
 */

public class Classifica {
    private List<Classificato> classificatiList;

    public Classifica(List<Classificato> classificatiList) {
        this.classificatiList = classificatiList;
    }

    public List<Classificato> getClassificatiList() {
        return classificatiList;
    }

    public void setClassificatiList(List<Classificato> classificatiList) {
        this.classificatiList = classificatiList;
    }
}

File: Deliverables/php/Entity/Classifica.php
<?php
	//**********************************************************************************************
    // Creazione dell' oggetto Classifica eredità Profilo che implementa JsonSerializable per essere inviato tramite json
    // Variabili: arrayClassificati
    // jsonSerialize() --> utilizzato per la conversione a json
    //**********************************************************************************************
	
    include ".../Entity/Profilo.php";
    include ".../Entity/Classificato.php";
    
    class Classifica implements JsonSerializable {
    	private $classificatiList;
        
        public function Classifica($classificatiList) {
        	$this->classificatiList = $classificatiList;
        }
        
        public function getClassificatiList(){
        	return $this->classificatiList;
        }
        
        public function setClassificatiList($classificatiList){
        	$this->classificatiList = $classificatiList;
        }
        
        public function jsonSerialize() {
        	return get_object_vars($this);
    	}
    }
?>